### PR TITLE
Fix defect #385

### PR DESCRIPTION
**Pull request by oGalceran1**
Fix defect #385

Implement also to indicate in the datepickertime component the option to reset the time in the spinners when setting the current date

### CODE REVIEW

File: projects/showcase/src/app/components/datepicker/showcase-datepicker.component.ts [hunk 1]
```diff
@@ -18,6 +19,13 @@ export class ShowcaseDatepickerComponent {
 		this.myDate = new Date(2018, 9, 20, 14, 5, 30, 0);
 		this.maxDate = new Date(2018, 9, 20);	// October 20, 2018
 		this.minDate = new Date(2017, 0, 20);	// January 20, 2017
+		this.myDateWithReset = new Date(2018, 9, 20, 14, 5, 30, 0);
 		this.isDisabled = false;
 	}
+
+	@ViewChild ('dateTimeWithReset', {static: false}) dateTimeWithReset: DatepickerTimeComponent;
```
Comment (carlosra85) [inline projects/showcase/src/app/components/datepicker/showcase-datepicker.component.ts:26]: Unused ViewChild¿?

File: projects/showcase/src/app/components/datepicker/showcase-datepicker.component.html [hunk 2]
```diff
@@ -13,6 +13,19 @@
                     [(currentDate)]="myDate"></systelab-date-time>
         </div>
     </div>
+    <div class="row mt-1">
+        <label class="col-md-3 col-form-label" for="form-h-s">Datepicker Time reseting time</label>
+        <div class="col-md-9">
+            <div class ="row ml-auto">
+                <systelab-date-time
+                        #dateTimeWithReset
```
Comment (carlosra85) [inline projects/showcase/src/app/components/datepicker/showcase-datepicker.component.html:21]: Unused ViewChild¿?

File: projects/systelab-components/src/lib/contextmenu/context-menu.component.ts [hunk 3]
```diff
@@ -45,22 +45,7 @@ export class ContextMenuComponent extends AbstractContextMenuComponent<ContextMe
...
-				this.previousActionId = actionId;
-
-				this.toggle(actionId + this.elementID);
-				const selectedChild = this.childDropdownMenuElement.toArray()
-					.find((elem) => elem.nativeElement.id === (actionId + this.elementID));
-				this.myRenderer.setStyle(selectedChild.nativeElement, 'top', this.getFirstChildTop(event, selectedChild) + 'px');
-				this.myRenderer.setStyle(selectedChild.nativeElement, 'left', this.getFirstChildLeft(selectedChild) + 'px');
-			}
-
+			this.doMouseOver(event, elementId, actionId);
```
Comment (carlosra85) [inline projects/systelab-components/src/lib/contextmenu/context-menu.component.ts:48]: Nice clean!

File: projects/showcase/src/app/components/datepicker/showcase-datepicker.component.ts (hunk 1 above)
Comment (oGalceran1) [reply projects/showcase/src/app/components/datepicker/showcase-datepicker.component.ts:26]: Thanks for your comment

File: projects/showcase/src/app/components/datepicker/showcase-datepicker.component.html (hunk 2 above)
Comment (oGalceran1) [reply projects/showcase/src/app/components/datepicker/showcase-datepicker.component.html:21]: Thanks for your comment

File: projects/systelab-components/src/lib/contextmenu/context-menu.component.ts (hunk 3 above)
Comment (oGalceran1) [reply projects/systelab-components/src/lib/contextmenu/context-menu.component.ts:48]: Thanks

**Pull request merged**